src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -560,4 +561,17 @@ public void repoConfig_removeIgnoredAuthors_success() throws Exception {
 
         TestUtil.compareRepoConfig(expectedConfig, actualConfig);
     }
+
+    @Test
+    public void repoConfig_invalidLocation_success() throws Exception {
```

Comment:
Assuming you are testing a case where cloning is failed? The naming should be sth like invalidLocation_fail or invalidLocation_throw...Exception

Reply:
Uh I am not sure about this because I am actually testing whether the invalid location will just be skipped and not added into the list of repositories that will be cloned and analyzed later. The expected behavior is that the exception will be caught by `getRepoConfigurations` and the method will skip the repository location accordingly